internal/saucecloud/cloud.go

```diff
@@ -333,6 +339,14 @@ func (r *CloudRunner) uploadProject(filename string, pType uploadType) (string,
 	return resp.ID, nil
 }
 
+func (r *CloudRunner) checkIfFileExists(fileName string) (string, error) {
```

Comment:
Could there be a better name for this? It's a bit unclear from the method's name & signature where it's going to look for this file's existence. If I were to read this method without any knowledge, I'd simply expect this method to verify whether the file `fileName` exists on the filesystem that it is pointing to (essentially a shorthand version for `os.Stat()`).